<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateCatForm</class>
 <widget class="QWidget" name="CreateCatForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QStackedWidget" name="CatStacked">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="CatPage1">
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="1">
        <widget class="QPushButton" name="CatCancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLineEdit" name="CatDirLine"/>
       </item>
       <item row="2" column="3">
        <widget class="QToolButton" name="CatDirButton">
         <property name="text">
          <string notr="true">...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QLabel" name="CatDirLabel">
         <property name="text">
          <string>Choose a directory for creating catalog from</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="2">
        <widget class="QPushButton" name="CatNextButton">
         <property name="text">
          <string>Create Catalog</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="CatPage2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <widget class="QPushButton" name="CatCloseButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QProgressBar" name="CatProgress">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="CatCreatingLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Catalog creating...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>CatDirLine</tabstop>
  <tabstop>CatDirButton</tabstop>
  <tabstop>CatCancelButton</tabstop>
  <tabstop>CatNextButton</tabstop>
  <tabstop>CatCloseButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>CatCancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CreateCatForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CatCloseButton</sender>
   <signal>clicked()</signal>
   <receiver>CreateCatForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
